server/app.py

```diff
@@ -68,8 +79,6 @@ def get_app(config=None):
         json_encoder=MongoJSONEncoder,
         validator=SuperdeskValidator)
 
-    superdesk.app = app
```

Comment:
The superdesk.app package variable is used a few(34) times in code. Probably we have to change the code and use the current application instead.